webui/react/src/constants/states.ts

```diff
@@ -24,9 +24,29 @@ export const activeRunStates: Array<
   'STATE_STOPPING_ERROR',
 ];
 
+const jobStates: Array<JobState> = [
+  JobState.QUEUED, JobState.SCHEDULED, JobState.SCHEDULEDBACKFILLED,
+];
+// DISCUSS maybe not combining the states would have been better as JobState is
+// a subset of RunState.Active
```

Comment:
I think this is good, it seems a bit more clear to me when I see naming like `pausableRunStates` and `cancellableRunStates`. We can adjust when need be once states get a clean up.